sqlmesh/core/config/root.py

```diff
@@ -111,3 +113,7 @@ def test_connection(self) -> ConnectionConfig:
         if isinstance(self.test_connection_, str):
             return self.get_connection(self.test_connection_)
         return self.test_connection_
+
+    @property
+    def dialect(self) -> t.Optional[str]:
```

Comment:
This could mean either dialect the model are written in, or the dialect of the target connection. Can we disambiguate?

Reply:
i thought it was only the model dialect